apps/src/blockly/addons/functionEditor.js

```diff
@@ -97,6 +97,7 @@ export default class FunctionEditor {
   hide() {
     if (this.dom) {
       this.dom.style.display = 'none';
+      this.clearEditorWorkspace();
```

Comment:
Curious: Is there a specific reason you added this on `hide` even though we already have it on `open`?

Reply:
Yep, if you were editing the description and didn't click out of the block, then hid the dom, the text input would still appear until you clicked somewhere else. It might be a blockly issue, but it's pretty easy to just clear the workspace to clean things up.

Reply:
Sounds reasonable! Should we mention it to Maribeth just in case they want to file a ticket?

Reply:
yep, updated!